pkg/monitoring/prometheusagent/secret.go

```diff
@@ -1,27 +1,65 @@
 package prometheusagent
 
 import (
+	"context"
 	"fmt"
+	"strings"
 
 	"github.com/pkg/errors"
 	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/yaml"
+)
 
-	"github.com/giantswarm/observability-operator/pkg/monitoring"
+const (
+	authSecretName      = "mimir-basic-auth"
```

Comment:
All the more reason to call this api key :D

Reply:
:shrug: